content/en/docs/quickstart.md

```diff
@@ -43,86 +43,144 @@ notation list --plain-http $IMAGE
 
-Confirm your container image fails validation. For example:
+For a Linux user, the directory is `$HOME/.config/notation/`.
 
-```output
-$ notation verify --plain-http $IMAGE
-Error: trust certificate not specified
-2022/09/30 15:24:14 trust certificate not specified
-```
+For a Window user, the directory is `C:/Users/AppData/Roaming/notation/`.
 
-This verification fails because you have not trusted the public key for your container image.
+For a Mac user, the directory is  `$HOME/Library/Application Support/notation/`
 
-## Add a trusted public key
+Once the trust policy file is executed as below, it creates one trust policy named `wabbit-networks-images`. The trust policy with `registryScopes` set to `*` applies to all the artifacts in any registries. The level of signatureVerification is `strict`, which enforces a full validation. Any validation failure will fail the whole signature verification process. The trust store used for this policy is named `wabbit-networks.io` of type `ca`, which is created in previous step. See [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy-properties) to fine tune the policies for specific security requirements.
 
-Add the public from the certificate you generated earlier as a trusted certificate.
+For linux user,
 
 ```console
-notation cert add --name "wabbit-networks.io" <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.crt
+cat <<EOF > "$HOME/.config/notation/trustpolicy.json"
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-images",
+            "registryScopes": [ "*" ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:wabbit-networks.io" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+}
+EOF
 ```
 
-Adding this public key as a trusted public key allows you to verify any container images that are signed with the corresponding private key. 
+For Mac user,
 
-```output
-$ notation verify --plain-http $IMAGE
-sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
+```console
+cat <<EOF > "$HOME/Library/Application Support/notation/trustpolicy.json"
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-images",
+            "registryScopes": [ "*" ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:wabbit-networks.io" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+}
+EOF
 ```
 
-The above example shows one trusted signature.
+For Windows user, create a json file named `trustpolicy.json` with the following content, and store it under `C:/Users/AppData/Roaming/notation/`
+
```

Comment:
I'd create one policy file in the doc and have different os version home directories in one spot